src/index.html

```diff
@@ -11,7 +11,103 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+      <fieldset class="field">
+        <legend>Personal information</legend>
+        <label class="form-field">
+          Surname
+          <input type="text" name="Surname"
+          autocomplete="off"
+          required>
```

Comment:
fix attributes in all places
```suggestion
          <input
            type="text"
            name="Surname"
            autocomplete="off"
            required
          >
```